website/other/bezier-rs-demos/src/App.vue

```diff
@@ -233,8 +220,8 @@ export default defineComponent({
 				},
 				{
 					name: "Project",
-					callback: (bezier: WasmBezierInstance, _: Record<string, number>, mouseLocation: Point): string =>
-						mouseLocation ? bezier.project(mouseLocation.x, mouseLocation.y) : bezier.to_svg(),
+					callback: (bezier: WasmBezierInstance, _: Record<string, number>, mouseLocation: number[]): string =>
```

Comment:
Replace `mouseLocation: number[]` with `[x, y]: [number, number]`

Reply:
This destructing notation seems to cause issues when we send an undefined mouseLocation to the project function, ie. on the initial render. Do you know of a good workaround @Keavon, or can we just leave it as is?

Reply:
We shouldn't be sending `undefined` if the declared type is `number[]` and if it's being read as `mouseLocation[0]` and `mouseLocation[1]` which would throw an exception for reading properties of `undefined`.